git_machete/tests/functional/test_machete.py

```diff
@@ -175,6 +182,20 @@ def setup_discover_standard_tree(self) -> None:
             """,
         )
 
+    def test_branch_reappers_in_definition(self) -> None:
+        body: List[str] = ['master\n', '    develop\n', '    \n', '    \n', 'develop\n']
```

Comment:
Pls include some empty lines as well! Also, this would read better as either list of strings split into multiple lines, or even better as a multiline string

Reply:
Also, are `\n`s at the end of each line intended? Aren't newline characters added automatically after each line by `writelines` method?

Reply:
I was also convinced that `\n` is added as separator but docs says something opposite. Nevertheless i changed it into multiline string.